polling_stations/apps/data_importers/management/commands/import_middlesbrough.py

```diff
@@ -3,58 +3,47 @@
-            "100110131622",  # FLAT 1 169 BOROUGH ROAD, MIDDLESBROUGH
-            "10023181264",  # FLAT 1 168 LINTHORPE ROAD, MIDDLESBROUGH
-            "10023172775",  # GRANGE LODGE, HEMLINGTON LANE, HEMLINGTON, MIDDLESBROUGH
-            "100110791007",  # GRANGE HOUSE, HEMLINGTON LANE, HEMLINGTON, MIDDLESBROUGH
-            "10093978033",  # FLAT 1 TOBY CARVERY MARTON ROAD, MIDDLESBROUGH
-            "10023181033",  # GROUND FLOOR 82 BOROUGH ROAD, MIDDLESBROUGH
-            "10023181034",  # FIRST FLOOR 82 BOROUGH ROAD, MIDDLESBROUGH
+            "10023174667",  # FLAT CAPTAIN COOK HOTEL 7-11 DURHAM STREET, MIDDLESBROUGH
+            "10023181276",  # FLAT 1 169 BOROUGH ROAD, MIDDLESBROUGH
+            "200000259465",  # 220A LINTHORPE ROAD, MIDDLESBROUGH
+            "100110140269",  # 30 THORNFIELD ROAD, MIDDLESBROUGH
+            "100110140271",  # 32 THORNFIELD ROAD, MIDDLESBROUGH
+            "100110109926",  # 44 FALKLAND STREET, MIDDLESBROUGH
+            "10093978798",  # FLAT 5 73 BOROUGH ROAD, MIDDLESBROUGH
+            "10093978797",  # FLAT 4 73 BOROUGH ROAD, MIDDLESBROUGH
+            "10023175713",  # FLAT NAVIGATION INN MARSH ROAD, MIDDLESBROUGH
         ]:
             return None
 
-        if record.addressline6 in ["TS1 3QD", "TS1 2ET", "TS5 5EG", "TS1 4AD"]:
+        if record.addressline6 in [
+            "TS5 5EG",  # splits
+            "TS1 3QD",  # VICTORIA ROAD, MIDDLESBROUGH
+            "TS1 4AD",  # 27 BOROUGH ROAD, MIDDLESBROUGH
+            "TS1 4AG",  # 149 LINTHORPE ROAD, MIDDLESBROUGH
+        ]:
             return None
 
         return super().address_record_to_dict(record)
 
     def station_record_to_dict(self, record):
-        # Memorial Hall Meldyke Lane Stainton Middlesbrough TS8 9AU
-        if record.polling_place_id == "9056":
-            record = record._replace(polling_place_easting="448034")
-            record = record._replace(polling_place_northing="514186")
-
-        # Marton Manor Primary School The Derby Middlesbrough TS7 8RH
-        if record.polling_place_id == "9044":
-            record = record._replace(polling_place_easting="451145")
-            record = record._replace(polling_place_northing="515806")
-
-        # ACTES Resource Centre Meath Street Middlesbrough TS1 4RY
-        if record.polling_place_id == "8923":
-            record = record._replace(polling_place_easting="448379")
-            record = record._replace(polling_place_northing="519367")
-
-        # Linthorpe Community School Roman Road Linthorpe Middlesbrough TS5 6EA
-        if record.polling_place_id == "8948":
-            record = record._replace(polling_place_easting="448699")
-            record = record._replace(polling_place_northing="518439")
+        # Mobile Station - Hemlington Grange Way, Junction of Ravensgill Road, Middlesbrough
+        if record.polling_place_id == "9525":
+            record = record._replace(polling_place_easting="0")
```

Comment:
This should really be `polling_place_easting=""`.  Luckily it doesn't geocode to the grid origin with `(0,0)`, but less confusing to have the values blank.